_data/opencollective.json

```diff
@@ -1040,14 +1044,14 @@
   {
     "name": "Open Collective",
     "last_payment": 0.0,
-    "all_time": 205.0,
+    "all_time": 210.0,
```

Comment:
This seems odd. Last payment is 0 but `all_time` increased?

Reply:
from what I see, Open Collective is not donating to us, and this is a copy of the one below. I don't know what's going on. I'll open the ticket, but let's merge this nevertheless.